src/bin/cargo.rs

```diff
@@ -186,6 +207,31 @@ fn execute(flags: Flags, config: &Config) -> CliResult<Option<()>> {
     Ok(None)
 }
 
+fn list_of_alias_commands(config: &Config, command: &String) -> Option<Vec<String>> {
```

Comment:
I think this'll want to return a `CargoResult` of the output so malformed configuration can be printed as well.